src/guide/jbs-jdk-bug-system.md

```diff
@@ -409,15 +657,21 @@ Examples:  If a bug fix only corrects a change in the build system, then add the
   <tr>
     <td class="dictionary">[[problemlist]{.jbs-label}]{#problemlist}</td>
     <td class="dictionary">
-      One or more tests has been problemlisted due to this bug.
+      One or more tests has been problem-listed due to this bug.
     </td>
   </tr>
   <!-- Q -->
   <!-- R -->
   <tr>
     <td class="dictionary">[[regression]{.jbs-label}]{#regression}</td>
     <td class="dictionary">
-      Used to identify regressions. A regression is a bug that didn't exist in the previous release. Ideally all regressions must be fixed in the following release. All regressions must have the [Affects Version/s]{.jbs-field} set.
+      Used to identify regressions. A regression is where behavior has _incorrectly_ changed from a previous build or release. Ideally all regressions must be fixed in the following release. All regressions must have the [Affects Version/s]{.jbs-field} set.
```

Comment:
> Ideally all regressions must be fixed in the following release

This sounds weird. Ideally we identify and fix the regressions before they are released. We don't wait until the following release to fix them.

Reply:
agreed, adjusted it to:
Ideally all regressions are identified and fixed before they are released, if not they must be fixed at latest in the following release after they are identified.